server/server.js

```diff
@@ -0,0 +1,43 @@
+const express = require('express');
+const app = express();
+const PORT = 3000;
+const httpProxy = require('http-proxy');
+const proxy = httpProxy.createProxyServer();
+
+const photos = 'http://localhost:3001';
+const itemSummary = 'http://localhost:3002';
+const relatedProducts = 'http://localhost:3003';
+const reviews = 'http://localhost:3004';
+
+
+app.use('/:id', express.static('./public'));
+
+app.all('/api/photos/:id', (req, res) => {
+  console.log('Connected to Photos!');
```

Comment:
I wonder if these logs should go after the proxy.web() invocation. Seems like they're preemptively logging a success before the attempt is made to connect